### PR TITLE
Update knobs to control: MetamaskTemplateRenderer

### LINKED ISSUE #18498: Update knobs to controls in storybook: MetamaskTemplateRenderer
### Description
Currently there are a few storybook stories still using knobs in our storybook files `*.stories.js`. knobs have been deprecated in favour of controls. It would be great to migrate these last stories from knobs to controls.

This issue is to migrate the `ui/components/app/metamask-template-renderer/metamask-template-renderer.stories.js` from knobs to controls

Screenshot 2023-04-06 at 4 34 17 PM

### Requirements
If you would like to take on this issue please comment but before asking to take on this issue there are a few requirements that will help complete this task:
- Get storybook up and running `yarn storybook`
- Some knowledge of storybook is not required but would be beneficial for this task. [storybook docs here](https://storybook.js.org/)

### Technical Details
- Migrate knobs to controls

### Acceptance Criteria
- knobs have been migrated to controls for listed story
- Take Before / After screenshots of the knobs and new controls and attach to the PR
- Tag @georgewrmarshall, @garrettbear, or @NidhiKJha in the PR to review

### PR DESCRIPTION
**Pull request by dj-pundir**
## Explanation

The knobs are deprecated so its in continuity with other tasks to change the knobs with controls. Following are changes to the component `metamask-template-renderer.stories.js`

- Change knob to control
- Remove `WithInvalidElement` story altogether 

Fixes:  https://github.com/MetaMask/metamask-extension/issues/18498

## Screenshots/Screencaps

Here is 

### Before
image

### After

image

## Manual Testing Steps

`yarn storybook`

- Go to Component -> App
- Select **MetamaskTemplateRenderer**

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [x] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

### CODE REVIEW

**Comment by dj-pundir** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ui/components/app/metamask-template-renderer/metamask-template-renderer.stories.js [hunk 1]
```diff
@@ -85,25 +91,12 @@ const SECTIONS = {
...
-      },
-      {
-        element: 'Unsafe',
-        key: 'unsafe-tree',
-        children:
-          'I should be displayed, but I wont be due to unsafe component',
-      },
-    ])}
-  />
-);
```
Comment (georgewrmarshall) [inline ui/components/app/metamask-template-renderer/metamask-template-renderer.stories.js:109]: I suggested to @dj-pundir to remove this story as it was display an error intentionally but I don't think it adds much value in storybook

**Review by georgewrmarshall — COMMENTED** [review]
Looking good looks like there is a linting issue that needs to be fixed

**Comment by dj-pundir** [comment]
> Looking good looks like there is a linting issue that needs to be fixed

Done the linting. Thanks!!

**Review by georgewrmarshall — APPROVED** [review]
LGTM! 💯 Thanks for your contribution!

**Review by brad-decker — APPROVED** [review]

**Pull request merged**